### PR TITLE
Make `tests/conversion` a bit faster

### PR DESCRIPTION
**Pull request by simondeziel**
Avoiding the 2 go snap installation should shave ~1m15s.

### CODE REVIEW

**Comment by simondeziel** [comment]
> Avoiding the 2 go snap installation should shave ~1m15s.

The savings were actually more like 3-4 minutes but I guess that's mostly dependent on the snap store being slow or fast.

The CI failures are not related.

File: tests/conversion [hunk 1]
```diff
@@ -255,7 +258,7 @@ IMAGE_PATH="${tmpdir}/backup/virtual-machine.img"
 # Test VM migration using conversion mode. If server does not support
 # conversion API extension, lxd-migrate must fallback to migration
 # mode and successfully transfer the VM disk.
-for driver in btrfs ceph dir lvm zfs; do
```
Comment (MusicDin) [inline tests/conversion:258]: Why is `ceph` removed?

**Review by MusicDin — APPROVED** [review]
LGTM, just one question :) Thanks!

File: tests/conversion (hunk 1 above)
Comment (simondeziel) [reply tests/conversion]: Cause I didn't bother setting up microceph for testing locally and accidentally scooped that edit in... thanks for catching!

Comment (simondeziel) [reply tests/conversion]: Oh incidentally, me accidentally dropping `ceph` from the tested storage driver probably explains why the tests were faster ... silly me!

**Review by tomponline — APPROVED** [review]

**Pull request merged**